### DISCUSSION TITLE
Use dependabot to automatically update and manage dependencies [13600]

### DISCUSSION THREAD

**Discussion by pablorcum** [eProsima/Fast-DDS#2438] [Ideas]
See more info [here](https://dependabot.com/).

**Comment by JLBuenoLopez** [comment]
Thanks for your suggestion @pablorcum!

I have just moved the issue to this forum where we want to keep community suggestions and improvements.

I want also to update the [dependabot link](https://github.com/dependabot) (as the one you provided just redirects to github.com).
